STYLEGUIDE.md

```diff
@@ -162,6 +173,18 @@ English speakers may find themselves using phrases which cultural meaning may be
 
 Some learners may not have heard this expression before. It may be difficult for machine translation as well.
 
+### 2.8 Use a active voice vs. passive voice.
+
+Use active verbs that put the reader in the first person instead of passive verbs ("be" verbs that describe the learners actions as a state of being like "is/was/to be").
```

Comment:
```suggestion
Use active verbs that put the reader in the first person instead of passive verbs; "be" verbs that describe the learners actions as a state of being, like "is/was/to be".
```